Clients/src/application/repository/entity.repository.ts

```
@@ -258,3 +258,31 @@ export async function getAllUsers({
+  try { 
+    const response = await apiServices.post(routeUrl, body, {
+      headers: { 
+        Authorization: `Bearer ${authToken}`,
+      },
+      signal,
+      responseType: 'blob'
+    });
+
+    return response;
+  } catch (error) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)